tests/integration/components/github-issue-select-test.js

```diff
@@ -0,0 +1,100 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import PageObject from 'ember-cli-page-object';
+import pageObject from '../../pages/components/github-issue-select';
+import Ember from 'ember';
+
+const { set, run } = Ember;
+
+const page = PageObject.create(pageObject);
+
+moduleForComponent('github-issue-select', 'Integration | Component | github issue select', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+    // defaults
+    set(this, 'issues', []);
+    set(this, 'onIssueSelected', () => {});
+    renderPage();
+  },
+  afterEach() {
+    page.removeContext();
+  }
+});
+
+function renderPage() {
```

Comment:
It actually would. I started the implementation thinking I'd be calling `renderPage()` for each test individually.